lib/gooddata.rb

```diff
@@ -31,3 +31,4 @@ module GoodData
 
 # Extensions
 require_relative 'gooddata/extensions/extensions'
+require 'backports/2.1.0/array/to_h'
```

Comment:
Writing and reading workarounds takes time. Is it better for rapid development to use `to_h` or something like this? `Hash[*keys.zip(vals).flatten]`